vite.config.js

```diff
@@ -20,6 +20,10 @@ export default defineConfig(() => {
     ],
     server: {
       port: 8081,
+      host: true,
+      watch: {
+        usePolling: true,
```

Comment:
Everything looks good! Just one small thing: according to the docs, `usePolling` uses high CPU utilisation so we should probably be using this for developer setup only and not production env? https://vitejs.dev/config/server-options#server-watch

Reply:
sounds good!